### PR TITLE
feat(docs): document feedback

### PR DESCRIPTION
**Pull request by chdeskur**
This PR adds documentation for the `edit-this-page` and `on-page feedback` features.

### CODE REVIEW

File: fern/pages/docs/getting-started/global-configuration.mdx [hunk 1]
```diff
@@ -303,3 +329,42 @@ layout:
...
+    owner: fern
+    repo: plant-store-docs
+    branch: main
+```
+
+<ParamField path="edit-this-page.github" type="object" required={false}>
+  Set the GitHub repository and branch where you would like users to suggest edits. 
+  
+  Learn more about the GitHub configuration [here](#github-configuration).
+</ParamField>
```
Comment (dsinghvi) [inline fern/pages/docs/getting-started/global-configuration.mdx:345]: i think we can actually directly add the github configuration here

File: fern/pages/docs/getting-started/global-configuration.mdx [hunk 2]
```diff
@@ -303,3 +329,42 @@ layout:
   and a 1px border will separate the sidebar from the content.
 </ParamField>
 
+## Edit this page configuration
+```yaml docs.yml
```
Comment (dsinghvi) [inline fern/pages/docs/getting-started/global-configuration.mdx:333]: can we highlight lines 2 to the end

File: fern/pages/docs/getting-started/global-configuration.mdx (hunk 1 above)
Comment (dsinghvi) [reply fern/pages/docs/getting-started/global-configuration.mdx:345]: so something like 

```
<ParamField path="github.owner" type="string" required={true}>
  The owner of the GitHub repository where you host your documentation.  
</ParamField>

<ParamField path="github.repo" type="string" required={true}>
  The name of the GitHub repository where you host your documentation.  
</ParamField>

<ParamField path="github.branch" type="string" required={true}>
  The branch of the repository you would like the GitHub editor to open a PR to. Default is `main`. 
</ParamField>
```

File: fern/pages/docs/getting-started/global-configuration.mdx (hunk 2 above)
Comment (chdeskur) [reply fern/pages/docs/getting-started/global-configuration.mdx:333]: Going to hold off since the rest of the code blocks aren't highlighted.

**Review by abvthecity — APPROVED** [review]

**Pull request merged**